Vagrantfile

```diff
@@ -52,7 +52,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder "/home/willmendesneto/projects/ng-box/projects", "/vagrant_data"
+  config.vm.synced_folder ENV['HOME'] + "/ng-box/projects", "/vagrant_data", create: true
```

Comment:
Great @gbrennon, this code is really incorrect. Remove the user is correct, but use `ENV['HOME']`, not. This code should be updated for the developer that will use the vagrant box. This code works on in *unix system operation, but doesn't works in Windows environment. It's good modify this code for `../vagrant-frontend-mobile-box/projects`, for example